packages/frontend/src/components/Game/GameBoard/TileGrid/index.test.jsx

```diff
@@ -2,6 +2,7 @@ import { setupTests } from 'helpers/tests';
 import { createTilesFromPhotos } from 'helpers/createTilesFromPhotos';
 import TileGrid from '.';
 import { mockPhotos } from '__mocks__/api/mockPhotos';
+import userEvent from '@testing-library/user-event';
```

Comment:
I believe that the `user` object that's returned from `setupTests` contains an instantiated version of `userEvent`, so this shouldn't need to be imported anywhere else.